data/src/main/java/com/teamagam/gimelgimel/data/message/rest/RestAPI.java

```diff
@@ -47,7 +46,7 @@ private void initializeMessagingAPI() {
 
         Retrofit retrofit = new Retrofit.Builder()
                 .baseUrl(Constants.MESSAGING_SERVER_URL)
-                .addCallAdapterFactory(RxJavaCallAdapterFactory.create())
+                .addCallAdapterFactory(RxErrorHandlingCallAdapterFactory.create())
```

Comment:
It's an error handle.
Retrofit throws fatal exception on failure, such as timeout or even bad status code (not 200).
We want to use the doOnError instead of catching exception from the observable.